src/main/java/reposense/util/FileUtil.java

```diff
@@ -38,6 +39,49 @@
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
+    /**
+     * Zips all the relative folders and relevant files of {@code fileTypes} contained in
+     * {@code sourceAndOutputPath} directory into the same folder.
+     */
+    public static void zipRelativeFiles(HashSet<Path> relativePaths,
+                                        Path sourceAndOutputPath, String... fileTypes) {
+        zipRelativeFiles(relativePaths, sourceAndOutputPath, sourceAndOutputPath, fileTypes);
+    }
+
+    /**
+     * Zips all the relevant files and relative folders
+     * @param relativePaths contains the relevant folders to be zipped.
+     * @param fileTypes contains the file types to be zipped. Only files which are of the type "fileTypes" will be
+     *                  zipped.
+     */
+    public static void zipRelativeFiles(HashSet<Path> relativePaths,
+                                        Path sourcePath, Path outputPath, String... fileTypes) {
```

Comment:
Same here, not following our coding style. We adopt 8 indentations from the previous line.